### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by MakaevaElena**
1. Task: [GraphiQL](https://github.com/rolling-scopes-school/tasks/blob/master/react/modules/graphiql.md)
2. Screenshot: 
image

3. Deploy: [graphiql-app]()
4. Done 08.12.2023 / Deadline 08.01.2024

### Footer
Footer 
- [x] should contain a link to the authors' GitHub, the year the application was created, [course logo](https://rs.school/images/rs_school_js.svg) with [link to the course](https://rs.school/react/).
- [x] Footer is displayed on all pages of the application.

### CODE REVIEW

File: src/Components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              variant="h6"
+              sx={{
+                color: '#eee',
+                fontStyle: 'normal',
+                textAlign: 'center',
+                fontSize: '1rem',
+                // verticalAlign: 'center',
+              }}
+            >
+              2024&copy; Up&GoTeam
```
Comment (MaleryValery) [inline src/Components/Footer/Footer.tsx]: can we change it to {new Date().getFullYear()}?

File: src/assets/img/github.png
Comment (MaleryValery) [inline src/assets/img/github.png:1]: MUI has their own build in icon, i took it for our cards in welcome page
 <Link href={gitHub} target="_blank">
    <GitHubIcon sx={{ color: '#eee' }} />
</Link>

File: src/Components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                <img
+                  alt="Rolling Scopes School Logo"
+                  className={styles['logo-img']}
+                  src="https://rollingscopes.com/images/logo_rs_text.svg"
+                />
+              </Icon>
+            </Link>
+            <Link href="https://github.com/MakaevaElena/graphiql-app">
+              <Icon sx={{ width: 50, height: 80 }}>
+                <img
```
Comment (MaleryValery) [inline src/Components/Footer/Footer.tsx]: lets use this one <GitHubIcon sx={{ color: '#eee' }} />

**Review by surface74 — APPROVED** [review]
ОК

File: src/Components/Footer/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+              gap: 2,
+              justifyContent: 'center',
+              alignItems: 'center',
+              // borderRadius: 4,
+              // border: 1,
+              // borderColor: '#eee',
+              p: 1,
+            }}
+          >
+            <Link href="https://rs.school/react/">
```
Comment (MaleryValery) [inline src/Components/Footer/Footer.tsx:20]: if we have course icon in footer i will remove it from welcome page(we have  requirement "The welcome page should contain general information about the developers, project, and course 10 points")

Comment (MakaevaElena) [reply src/Components/Footer/Footer.tsx:20]: course icon  must be used  in footer as required by TS

File: src/Components/Footer/Footer.tsx (hunk 2 above)
Comment (MakaevaElena) [reply src/Components/Footer/Footer.tsx]: ок, I used it

File: src/assets/img/github.png
Comment (MakaevaElena) [reply src/assets/img/github.png:1]: ок, I used it

File: src/Components/Footer/Footer.tsx (hunk 1 above)
Comment (MakaevaElena) [reply src/Components/Footer/Footer.tsx]: great idea

**Review by MaleryValery — APPROVED** [review]

**Review by surface74 — APPROVED** [review]
OK

File: src/Components/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                  className={styles['logo-img']}
+                  src="https://rollingscopes.com/images/logo_rs_text.svg"
+                />
+              </Icon>
+            </Link>
+            <Link href="https://github.com/MakaevaElena/graphiql-app">
+              <GitHubIcon sx={githubIcon} />
+            </Link>
+            <Typography variant="h6" sx={yearTitle}>
+              {new Date().getFullYear()}&copy; Up&GoTeam
```
Comment (AlexSerdyuk83) [inline src/Components/Footer/Footer.tsx]: стоит вынести в переменную

Comment (MakaevaElena) [reply src/Components/Footer/Footer.tsx]: ок, вынесла

**Pull request merged**